packages/block-editor/src/layouts/flow.js

```diff
@@ -125,6 +133,8 @@ export default {
 
 					${ appendSelectors( selector, '> [data-align="full"]' ) } {
 						max-width: none;
+						margin-left: calc( -1 * ${ padding.left || 0 } ) !important;
```

Comment:
I think these also should be optionally chained? 
```suggestion
						margin-left: calc( -1 * ${ padding?.left || 0 } ) !important;
```